### PR TITLE
Fixed broken user language setting workflow

### PR DESCRIPTION
**Pull request by themoonraker13**
The fix is for the issue discussed in: https://issues.openmrs.org/browse/RA-997

With the code changes, the user's default language/locale shall be in effect once the user has successfully logged into the system/application.

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java [hunk 1]
```diff
@@ -39,10 +37,11 @@
...
 
 import javax.servlet.http.HttpServletRequest;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Locale;
 
-import static org.openmrs.module.referenceapplication.ReferenceApplicationWebConstants.COOKIE_NAME_LAST_SESSION_LOCATION;
-import static org.openmrs.module.referenceapplication.ReferenceApplicationWebConstants.REQUEST_PARAMETER_NAME_REDIRECT_URL;
-import static org.openmrs.module.referenceapplication.ReferenceApplicationWebConstants.SESSION_ATTRIBUTE_REDIRECT_URL;
+import static org.openmrs.module.referenceapplication.ReferenceApplicationWebConstants.*;
```
Comment (djazayeri) [inline omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java]: Can you please change the IntelliJ setting to not do wildcard imports? We avoid these in OpenMRS. (Except perhaps in unit tests with import assert.*

File: omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java [hunk 2]
```diff
@@ -3,19 +3,16 @@
  * Version 1.0 (the "License"); you may not use this file except in
  * compliance with the License. You may obtain a copy of the License at
  * http://license.openmrs.org
- *
+ * <p>
```
Comment (djazayeri) [inline omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java]: The license text should not be reformatted.

File: omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java (hunk 1 above)
Comment (themoonraker13) [inline omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java]: Sure @djazayeri.

File: omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java (hunk 2 above)
Comment (themoonraker13) [inline omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java]: IntelliJ IDEA seems to be making the formatting related changes automatically. I need to disable it and then push the code.

**Pull request closed without merging**